### ISSUE TITLE
[Bug]: Color of the text in the navbar is light

### ISSUE THREAD

**Issue by TusharBhatt1** [piyushgarg-dev/piyushgargdev-nextjs#209]
### Describe the bug

Navbar text (Home, Courses, My Gears, Blog, Login) is light in color, as it is one of the heading, white color will look good.

### To Reproduce

Go to Home page

### Expected behavior

Navbar text should be white in color.

### Please checkmark the following checklist

- [X] I make sure that similar issue is not opened

**Comment by Pratyakshshiv** [comment]
can you assign this issue to me

**Comment by pragatiailani** [comment]
I think they pretty decent. Can make 'em a bit lighter but not proper white.

**Comment by akshitdevelper** [comment]
@pragatiailani i agree with your point. However, if we consider other elements of the navbar they are of white color and due to the minimalism of the navbar it is suggested to use atmost 3 colors in the navbar which is already there. white text, green initials and the background. 

 It would be more appealing if text is white as well.
